twilight-gateway/src/shard.rs

```diff
@@ -662,6 +662,7 @@ impl Shard {
                         .as_mut()
                         .map_or(false, |handle| Pin::new(handle).poll(cx).is_ready())
                 {
+                    self.identify_handle = None;
```

Comment:
This duplicates L775. You can keep whichever you feel is a better place for it.

Reply:
I must have missed this. Thanks for pointing it out. It should be fine to set it to None in the action block, instead of the poll. It's also matching the context better in its original place.